releasenotes/notes/handle-merging-new-secrets-0e1fd06d86f0abbd.yaml

```diff
@@ -0,0 +1,19 @@
+---
+features:
+  - The update-yaml.py script now contains an optional
+    argument, --output-file, for specifying an output
+    file.
```

Comment:
Small nit. It might be nice to get `update-yaml.py` and `--output-file` monospaces so they're easier to see.